hbase-common/src/saveVersion.sh

```diff
@@ -71,7 +71,7 @@ import org.apache.yetus.audience.InterfaceAudience;
 
 @InterfaceAudience.Private
 public class Version {
-  public static final String version = "$version";
+  public static final String version = new String("$version");
```

Comment:
Why make this change?

Reply:
Isn't it a String already?

Reply:
Because String can not be modified, I used setFinalStatic to set the version.